docs/reference/agent-config.txt

```diff
@@ -0,0 +1,45 @@
+.. _agent-configuration:
+
+Agent Configuration
+===================
+
+For both the master and the agent, configuration options may be set using a
+configuration file, environment variables, or command-line options. (Although
+values from different sources will be merged, we generally recommend sticking
+to a single source for each service to keep things simple.)
+
+Each option in the configuration file corresponds directly to an
+environment variable or command-line option. For example, the agent
+configuration file might contain
+
+.. code:: yaml
+
+  master_address: 0.0.0.0
+  master_port: 8080
+
+
+Required Options
+----------------
+The following options are required for connecting an agent to the Determined
+master.
+
+Master Address
```

Comment:
Using sub-sections here seems a bit overly verbose. How about a bulleted list with two elements, one for address and one for port? This is also kinda redundant with the complete list of config options below.